settings.gradle.kts

```diff
@@ -48,7 +48,7 @@ include(
 include("iceberg:iceberg-common")
 include("iceberg:iceberg-rest-server")
 include("authorizations:authorization-ranger")
-include("trino-connector")
+include("trino-connector", "trino-connector:integration-test")
```

Comment:
Why do we choose to moe integration to "trino-connector" as a submodule, not put them into src/test?

Typically, for a module, there will be "/src" under that module, but here it has another module under this module, it's not a common practice, I feel a little weird about this.

Reply:
> Why do we choose to move integration to "trino-connector" as a submodule, not put them into src/test?

It's a workaround for dependency conflicts. 

I moved the integration-test of Trino as a submodule because putting it into `src/test` caused a conflict between Trino testing (jetty11) and MiniGravitino (jetty9). Jetty11 is built for Java11 at a minimum, but MiniGravitino needs to be built on Java8.

> Typically, for a module, there will be "/src" under that module, but here it has another module under this module, it's not a common practice, I feel a little weird about this.

Adding a module directly is the smallest change. You can check if it's necessary to reorganize to the following directory structure:
```
trino-connector
├── trino-connector
│   ├── src
│   ├── build.gradle.kts
├── integration-test
│   ├── src
│   ├── build.gradle.kts
└── build.gradle.kts
```

Reply:
I'm inclined to use this style.
 
> Adding a module directly is the smallest change. You can check if it's necessary to reorganize to the following directory structure:
> 
> ```
> trino-connector
> ├── trino-connector
> │   ├── src
> │   ├── build.gradle.kts
> ├── integration-test
> │   ├── src
> │   ├── build.gradle.kts
> └── build.gradle.kts
> ```